scripts/ci/install-deps

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Don't use sudo when running in a container.
+SUDO=sudo
+if [ -f /.dockerenv ]; then
+  SUDO=
+fi
+
+# DISTRO_TYPE is either "upstream" or "openshift"
+DISTRO_TYPE_UPSTREAM="upstream"
+DISTRO_TYPE_OPENSHIFT="openshift"
+DISTRO_TYPE="${1:-$DISTRO_TYPE_UPSTREAM}"
+if [[ "$DISTRO_TYPE" != "$DISTRO_TYPE_UPSTREAM" && "$DISTRO_TYPE" != "$DISTRO_TYPE_OPENSHIFT" ]]; then
+  echo "DISTRO_TYPE \"$DISTRO_TYPE\" is not valid. Must be one of: \"$DISTRO_TYPE_UPSTREAM\", \"$DISTRO_TYPE_OPENSHIFT\"."
+  exit 1
+fi
+
+# jq, yq
+curl -Lo jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+chmod +x jq
+$SUDO mv jq /usr/local/bin/
+curl -Lo yq https://github.com/mikefarah/yq/releases/download/2.2.1/yq_linux_amd64
+chmod +x yq
+$SUDO mv yq /usr/local/bin/
+# SDK
+# TODO(estroz): use curl download method once v0.6.0 is released, which contains scorecard updates.
+# SDK_VER="v0.6.0"
+# curl -Lo operator-sdk "https://github.com/operator-framework/operator-sdk/releases/download/${SDK_VER}/operator-sdk-${SDK_VER}-x86_64-linux-gnu"
+SDK_REPO_PATH="github.com/operator-framework/operator-sdk"
+go get -d "$SDK_REPO_PATH" || true
```

Comment:
Why is this "or true" here? For example, if `go get` fails can we still use the operator SDK?

Reply:
For example, the test Travis output you've shown gives the following result after running `go get`: https://travis-ci.com/estroz/community-operators/builds/103568313#L245